blocks/photo-carousel/photo-carousel.css

```diff
@@ -60,8 +60,10 @@
   }
 
   .photo-carousel .photo-slides img {
-    width: 900px;
-    height: 600px;
+    min-width: 900px;
```

Comment:
Just curious, what's the difference between these 2 styles and how does it address the problem.

Reply:
The `width` property seems to be more of a hint and the browser actually makes the image smaller sometimes than specified with `width` which can be observed on the 'before' branch https://main--sunstar-foundation--hlxsites.hlx.live/en/education/jsdei/gallery/288 (just look at the right hand side of the picture in the carousel)

The `min-width` property prevents that. I think we still may need the `width` property to indicate that the image shouldn't grow larger either. Maybe `max-width` could be used instead of just `width` but I haven't seen the need for that. If you think `max-width` would be better than `width` I can change that.